src/client/pages/Debugger/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const ActualDebugger = React.lazy(() =>
+  import(/* webpackChunkName: "debugger" */ './ActualDebugger').then((m) => ({
+    default: m.ActualDebugger,
+  })),
+)
+
+export const Debugger: React.FC = () => {
+  const [mounted, setMounted] = React.useState(false)
+  React.useEffect(() => setMounted(true), [])
```

Comment:
Getting a bit confused by this `useEffect` 🤔 I thought the linter would complain about `setMounted` being absent from the dependency array? And won't `setMounted(true)` also be implicitly returned here, which may not be what we want since it will then be used as cleanup function? Or maybe that's not how it works 🤷‍♀️

Reply:
or maybe the linter wouldn't complain since I apparently didn't add `eslint-plugin-react-hooks` hehe 🤷‍♀️